src/jetstream.rs

```diff
@@ -99,7 +99,7 @@
 //! assert!(consumer_res.is_err());
 //!
 //! // this will create the consumer if it does not exist already
-//! let consumer = js.create_or_bind("my_stream", "existing_or_created_consumer")?;
+//! let consumer = js.subscribe("my_stream", "existing_or_created_consumer")?;
```

Comment:
Subscribe should always take a subject as first arg, so if this is still trying to bind to the stream name this is mis-named.